content/post-content.inc

```diff
@@ -1,4 +1 @@
-    </div> <!-- col -->
-  </div> <!-- row -->
-</div> <!-- container -->
```

Comment:
pulled out to `index.php`. was already included in `news/wp-content/themes/cmubuggy-child/footer.php`